Contributing/How-to-Contribute-Code.md

```diff
@@ -1,27 +1,42 @@
-## Code Contributions  
-
-1. Get to know moja global (Thank you for doing this already! Otherwise you would not be here.)  
-    * Read the [Contributing document](https://github.com/moja-global/About-moja-global/blob/master/CONTRIBUTING.md)
-    * Read the [Code of Conduct](https://github.com/moja-global/.github/blob/master/CODE_OF_CONDUCT.md)
-    * Review the [Coding Guidelines](https://github.com/moja-global/.github/blob/master/Governance/Coding-Guidelines.md) and the [Contribution Criteria](https://github.com/moja-global/.github/blob/master/Governance/Contribution-Criteria.md).
-    * Review the bug-list, open issues, Project Board (if you want to code a new feature) or the [Strategic Plan](https://github.com/moja-global/About-moja-global/blob/master/Governance/Strategic-Plan.md) (if you want to start a [new Project](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Start-a-New-Project.md).) 
-    * Contact the Code Coach if available in the README document or the Repo Maintainer by submitting an [Issue](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Provide-User-Feedback.md) with an @mention.  
-
-1. [Join the moja global organisation](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Join-moja-global.md)  
-
-1. [Fork the repository](https://help.github.com/en/articles/fork-a-repo) if you do not want to join moja global or want to start working before your request to join has been approved.
-
-1. Follow [Vincent Driessen's Git Branching](https://nvie.com/posts/a-successful-git-branching-model/) System
-    * Name your Branch with same name using the Feature or Branch name with a [version number](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Assign-a-Version.md)
-1. Make your contribution
-    * Make changes or additions to the code
-    * Follow [documentation rules](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Document-Your-Contribution.md)
-1. Submit your contribution
-    * [Submit a Pull-Request](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Submit-a-Pull-Request.md) to the Branch you want to contribute .
-    * Use a clear and descriptive title
-    * Ensure maintainers can understand your proposed changes from the description
-    * Sign your message with `Signed-off-by: Random J Developer <random@developer.example.org>` to agree to [Developer Certificate of Origin (DCO)](https://developercertificate.org/) (to certify that you wrote or otherwise have the right to submit your contribution to the project.)
```

Comment:
Maybe it is possible to automate DCO with a GitHub bot?

Reply:
It's in the goal as of now. I will integrate as soon as this PR gets merged on the repository.